.github/workflows/build-test-javascript.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         name: semgrep-js-ocaml-build-${{ github.sha }}
   test:
     runs-on: ubuntu-latest-16-core
-    container: emscripten/emsdk:3.1.46
+    container: emscripten/emsdk:3.1.51
```

Comment:
Nope, needs to modify the .jsonnet!

Reply:
We really need to make a pre-commit hook for this...